initial_posture: [0.0, 0.5, 2.1, 0.0, 0.0]
waypoint_pre: [0.0, 0.0, -0.07, 0.0, 0.0, 0.0]
waypoint_man: [0.0, 0.0, 0.04, 0.0, 0.0, 0.0]

